examples/2-bigger-project/README.md

```diff
@@ -147,7 +147,7 @@ let
   rustPkgs = pkgs.rustBuilder.makePackageSet' {
     rustChannel = "stable";
     packageFun = import ./Cargo.nix;
-    packageOverrides = pkgs: pkgs.rustBuilder.overrides.all;
+    # packageOverrides = pkgs: pkgs.rustBuilder.overrides.all; # Implied, if unspecified
```

Comment:
Unfortunately the default value for `packageOverrides` is an empty list. I wanted to make it explicit so when a native dependency is missing, users know where to look for. Do you think we should make `all` the default instead?

Reply:
Ahh, I see what you mean. I guess we do have to specify it instead. I think we should have it default to `all`, though, since that improves the user experience considerably. 👍

Nix is lazy, so it's not like it will attempt to build any of it unless it needs to. It also doesn't increase the fetch size of the overlay since we're already cloning the entire `cargo2nix` repository with `fetchTarball`/`fetchGit` anyway.

If the user wants to build with absolutely no default overrides, which I imagine a select few users might, they could provide:

```
packageOverrides = _: [];
```

Perhaps we could merge this PR first and then create a follow-up PR changing the default value?

Reply:
Yeah let's do that!